test/e2e/fixtures.go

```diff
@@ -128,20 +128,16 @@ func setupTest(tb testing.TB) (*TestData, error) {
 	return testData, nil
 }
 
-func setupTestWithIPFIXCollector(tb testing.TB) (*TestData, error, bool) {
-	// TODO: remove hardcoding to IPv4 after flow aggregator supports IPv6
-	// Also use setupTest.
-	isIPv6 := false
-	if err := testData.setupLogDirectoryForTest(tb.Name()); err != nil {
-		tb.Errorf("Error creating logs directory '%s': %v", testData.logsDirForTestCase, err)
-		return nil, err, isIPv6
-	}
-	tb.Logf("Creating '%s' K8s Namespace", testNamespace)
-	if err := ensureAntreaRunning(tb, testData); err != nil {
-		return nil, err, isIPv6
+func setupTestWithIPFIXCollector(tb testing.TB) (*TestData, bool, error) {
+	var isIPv6 bool
+	if clusterInfo.podV6NetworkCIDR == "" {
```

Comment:
networkPolicyOnly mode does not support dual stack. That has to be handled differently.
See this: https://github.com/vmware-tanzu/antrea/blob/e92ee627c2a218150935e41a62d43ed5a03e06b4/pkg/agent/config/node_config.go#L120

Reply:
Thanks.